Meta/Lagom/Tools/CodeGenerators/LibWeb/WrapperGenerator/IDLGenerators.cpp

```diff
@@ -2835,6 +2848,7 @@ void generate_constructor_implementation(IDL::Interface const& interface)
 #include <LibJS/Heap/Heap.h>
 #include <LibJS/Runtime/GlobalObject.h>
 #include <LibJS/Runtime/IteratorOperations.h>
+#include <LibJS/Runtime/TypedArray.h>
```

Comment:
Why? 🤔

Reply:
It failed the build of I removed it. I’ll retry a bulls tomorrow without the include

Reply:
Switched it out with `ArrayBuffer.h`. Was messing around with `JS::TypedArrayBase` earlier that's why I ended up with `TypedArray.h`.